src/SwiftUI/SwiftUI/Color.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	using static Color;
+
+	public enum RGBColorSpace
+	{
+		DisplayP3,
+		RGB,
+		RGBLinear,
```

Comment:
Hmm I think we should match the "s" prefix for the values from https://developer.apple.com/documentation/swiftui/color/rgbcolorspace, and I think `sRGB` should be the first value so it is the default :
```suggestion
		sRGB,
		DisplayP3,
		sRGBLinear,
```